base/features/data-hibernate-jpa/skeleton/src/main/resources/application.yml

```diff
@@ -0,0 +1,15 @@
+datasources:
+  default:
+    url: jdbc:h2:mem:devDb;MVCC=TRUE;LOCK_TIMEOUT=10000;DB_CLOSE_ON_EXIT=FALSE
+    driverClassName: org.h2.Driver
+    username: sa
+    password: ''
+    schema-generate: CREATE_DROP
```

Comment:
This applies only to the JDBC impl

Reply:
removed

Reply:
this was a leftover from something i tried earlier i missed, it has been removed